src/components/training/TrainingItem.js

```diff
@@ -1,30 +1,53 @@
 import React from 'react'
 import styled from 'styled-components'
 import { P } from '../text'
-import { Col } from '../layout/Grid'
+import { Col, Row } from '../layout/Grid'
 import { LinkButton } from '../buttons'
 import Link from '../navigation/Link'
 import { Image } from '../elements'
+import { selectTypeColor } from '../utils';
 
 const TrainingItemCol = styled(Col)`
   padding-bottom: 16px;
 `
 
-const TrainingItem = ({ name, city, country, startDate, path, imageSrc }) => (
+const TrainingRow = styled(Row)`
+  margin-bottom: 1em;
+`
+
+const Calander = styled.div`
+  border: 3px solid ${props => selectTypeColor(props.type)};
+  padding: 10px;
+  font-family: barlow;
```

Comment:
maybe here we can use de `FONT_FAMILY` styles config function?? at the end this will be part of the theme so...